app/uk/gov/hmrc/vatapi/resources/VatReturnsResource.scala

```diff
@@ -84,6 +94,11 @@ object VatReturnsResource extends BaseResource {
           }
         }
       }
+
+      result.recover {
+        case ex => logger.warn(ex.getMessage)
```

Comment:
Need to change the logging message to follow our agreed logging format: 

`logger.warn(s"[VatReturnsResource] [retrieveVatReturns] Unexpected error thrown on the call to ${DES: ex.getMessage}")`